dev/conformance/conformance-tests/set-st-alone-mergeall.json

```diff
@@ -17,6 +17,11 @@
                 "fields": {},
                 "name": "projects/projectID/databases/(default)/documents/C/d"
               },
+              "updateMask": {
```

Comment:
This needs to be an empty mask as you are otherwise deleting field "a" before applying the transform, which invalidates increment() and arrayUnion().

Reply:
Resolved. I think it's in a happy state now.

Reply:
Resolved. Also added a system-test for this.